src/libcrun/linux.c

```diff
@@ -244,17 +244,13 @@ static unsigned long
 get_mount_flags_or_option (const char *name, int current_flags, unsigned long *extra_flags, char **option)
 {
   int found;
-  cleanup_free char *prev = NULL;
```

Comment:
I put it back with a note saying you need it there, and asking for follow-up. Nothing else in that function calls *prev; but I'm assuming you're having it wipe here in the static function because of something else using it.